proto/reaction.proto

```diff
@@ -887,6 +896,7 @@ message ProductCompound {
   }
   Texture texture = 5;
   // Any features (e.g., calculated descriptors) of the product compound.
+  // Convention: also put stoichiometry ratio/coefficient here, see https://github.com/open-reaction-database/ord-schema/issues/683
```

Comment:
same comment as above; the proto definition probably isn't the right place for this